avocado/plugins/runner_nrunner.py

```diff
@@ -56,11 +56,11 @@ def initialize(self):
 
         help_msg = ('If the status server should automatically choose '
                     'a "status_server_listen" and "status_server_uri" '
-                    'configuration. Default is not to auto configure a '
+                    'configuration. Default is to auto configure a '
                     'status server.')
         settings.register_option(section=section,
                                  key='status_server_auto',
-                                 default=False,
+                                 default=True,
```

Comment:
I kept the two apart, because I felt like the switch to auto mode, would be more controversial than the introduction of the feature itself.

But, seems I was wrong.  I'll squash them on v2.